CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ community.
 ### Participation Conduct
 
 In order to ensure everyone has a fair, pleasant, and inclusive experience contributing to Neutrino, we ask that you
-abide by our [community participation guidelines](/contributing/code-of-conduct.md), based on the
+abide by our [community participation guidelines](/docs/contributing/code-of-conduct.md), based on the
```

Comment:
@eliperelman `/docs/contributing/README.md` is the file that the gitbook reads and in there the paths are relative. Is `CONTRIBUTING.md` also read by the gitbook? I don't see that link in `/docs/SUMMARY.md`.